identity-and-access-management/iam/api-cli/using-api-key-object-storage.mdx

```diff
@@ -22,10 +22,10 @@ In this document, we explain the concept of preferred Projects for Object Storag
 
 ## What is a Project?
 
-You can create multiple Projects within an Organization, to enable you to group your resources. Each Organization has at least one Project, called `default`, to which all resources you create (Instances, Kubernetes Kapsules, Object Storage buckets, etc.) are added if no other Projects are created.
+You can create multiple Projects within an Organization, to enable you to group your resources. Each Organization has at least one default Project to which all resources you create (Instances, Kubernetes Kapsules, Object Storage buckets, etc.) are added if no other Projects are created.
     <Lightbox src="scaleway-projects-1.webp" alt="" />
 
-When you create new Projects, you can choose which Project to add resources to, or manage resources in. However, Object Storage has its own limitations which make it harder to manipulate Projects, **specifically via an API**. When carrying out actions on Scaleway Object Storage resources via the Scaleway console, the target Project can be still selected from the Project dashboard or drop-down menu.
+When you create new Projects, you can choose which Project to add resources or manage resources in. However, Object Storage has its own limitations which make it harder to manipulate Projects, **specifically via an API**. When carrying out actions on Scaleway Object Storage resources via the Scaleway console, the target Project can be still selected from the Project dashboard or drop-down menu.
```

Comment:
I think the extra "to" is necessary!

Reply:
Or else "you can choose the Project in which you want to add or manage resources"

Reply:
```suggestion
When you create new Projects, you can choose the Project in which you want to add or manage resources. However, Object Storage has its own limitations which make it harder to manipulate Projects, **specifically via an API**. When carrying out actions on Scaleway Object Storage resources via the Scaleway console, the target Project can be still selected from the Project dashboard or drop-down menu.
```